projects/core/src/auth/user-auth/services/auth-config.service.ts

```diff
@@ -23,7 +23,7 @@ export class AuthConfigService {
    * @return client_id
    */
   public getClientId(): string {
-    return this.authConfig.authentication.client_id ?? '';
+    return this.authConfig?.authentication?.client_id ?? '';
```

Comment:
what about a little utility method to avoid all the lengthy optional chaining? 

```ts
constructor(
    protected config: AuthConfig,
    protected occConfig: OccConfig
) {}

private get authConfig(): AuthConfig['authentication'] {
    return this.config?.authentication ?? {};
}
```